web/static/js/reducers/survey.js

```diff
@@ -123,7 +123,8 @@ const validateFallbackDelay = (state: DataStore<Survey>) => {
   }
 }
 
-const timeSpecRegex = /^\d+[mhd]$/
+// Attempts are at least every 10m
+const timeSpecRegex = /^([1-9]\d+m)|([1-9]\d*[hd])$/
```

Comment:
This is abusing a bit the `10` value.

Refactoring the `timeSpecRegex.test` calls to use Regex for matching  `/^\d+[mhd]$/` and then `parseInt` to extract the value of the `\d+` using match data would be less obscure.

Reply:
I've just improved the code expressiveness. Please take a look now and let me know. Thanks!